generated/nidigital/nidigital/session.py

```diff
@@ -616,7 +1009,7 @@ def apply_tdr_offsets(self, offsets):
         nidigital.Session repeated capabilities container, and calling this method on the result.
 
         Args:
-            offsets (basic sequence of hightime.timedelta, datetime.timedelta, or float in seconds):
+            offsets (basic sequence of hightime.timedelta, datetime.timedelta, or float in seconds): TDR offsets to apply, in seconds. Specify an offset for each pin or channel in the **channelList**. If the **channelList** contains pin names, you must specify offsets for each site in the channel map per pin.
```

Comment:
I changed it to refer to slight variations of "repeated capabilities" because that seemed more consistent with how that concept is referred to in other places.